pennylane/ops/channel.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the available built-in noisy
+quantum channels supported by PennyLane, as well as their conventions.
+"""
+# pylint:disable=abstract-method,arguments-differ,protected-access
+
+import numpy as np
+
+from pennylane.operation import AnyWires, Channel
+
+
+class QubitChannel(Channel):
+    r"""QubitChannel(K_list, wires)
+    Apply an arbitrary fixed quantum channel.
+
+    Kraus matrices that represent the fixed channel are provided
+    as a numpy array.
+
+    **Details:**
+
+    * Number of wires: Any (the operation can act on any number of wires)
+    * Number of parameters: 1
+    * Gradient recipe: None
+
+    Args:
+        K_list (array(array[complex])): Array of Kraus matrices
```

Comment:
My suggestion would be either of the following two:

* Stick with passing lists to QubitChannel and create a new `par_domain` entry if need be: having array nesting `array(array[complex])` becomes complicated and infeasible at times and can have gotchas as the error that arises
* Simply use `return self._kraus_matrices(*self.data)` instead of `return self._kraus_matrices(*self.parameters)` for defining the `kraus_matrices` property for `Channel` --- this disallows training (`self.parameters` can contain `Variable` instances where `self.data` only contains numerical values). Not sure if training based on the input Kraus matrices or any of their parameters would even be a use case